### PR TITLE
UX-772/Added hostagent download links for DDUs

### PR DESCRIPTION
**Pull request by knnguy**

### CODE REVIEW

File: src/app/core/components/DownloadHostAgentWalkthrough.tsx [hunk 1]
```diff
@@ -21,11 +26,22 @@ enum OsDownloadLabel {
...
-enum osDownloadLinkName {
+enum OsDownloadLinkName {
   Linux = 'rpm_install',
   Ubuntu = 'deb_install',
 }
 
+const deccoDownloadOptions = [
+  {
+    label: OsDownloadLabel[OsOptions.Linux],
+    link: 'https://pmkft-1584219278-18238.platform9.io/clarity/platform9-install-redhat.sh',
```
Comment (jruddell) [inline src/app/core/components/DownloadHostAgentWalkthrough.tsx:37]: this is a hardcoded URL. Please think about this more before writing out a url like this. Its a product that will run on many different origins, this part should be dynamic

Comment (knnguy) [reply src/app/core/components/DownloadHostAgentWalkthrough.tsx:37]: Oops. Sorry, I misunderstood what you said about the links in Slack

File: src/app/core/components/DownloadHostAgentWalkthrough.tsx [hunk 2]
```diff
@@ -85,23 +101,31 @@ const DownloadHostAgentWalkthrough = ({ osOptions }): JSX.Element => {
   const classes = useStyles({})
   const [anchorEl, setAnchorEl] = useState(null)
   const [downloadOptions, setDownloadOptions] = useState([])
+  const session = useSelector<RootState, SessionState>(prop(sessionStoreKey))
+  const { features } = session
+  const isDecco = pathStrOr(false, 'experimental.kplane', features)
```
Comment (jruddell) [inline src/app/core/components/DownloadHostAgentWalkthrough.tsx:106]: Lets make this (isDecco) a helper as its used in more than one file

Comment (knnguy) [reply src/app/core/components/DownloadHostAgentWalkthrough.tsx:106]: I put it in a new helper file in `app/core/utils/helpers.ts`. Not sure if this is the best place to put it.

**Review by jruddell — APPROVED** [review]
can you also update the other places where we check to see if the environment is a decco one to use this helper?

**Pull request merged**